locale/nb_NO/LC_MESSAGES/django.po

```diff
@@ -43,311 +43,501 @@ msgstr "norlønn-rapport"
 msgstr "logo"
 
-#: spbm/apps/society/models.py:44 spbm/apps/society/models.py:233
+#: spbm/apps/society/models.py:47 spbm/apps/society/models.py:277
 msgid "worker"
 msgstr "arbeider"
 
-#: spbm/apps/society/models.py:45
+#: spbm/apps/society/models.py:48
 msgid "workers"
 msgstr "arbeidere"
 
-#: spbm/apps/society/models.py:56
-msgid "active"
-msgstr "aktiv"
-
-#: spbm/apps/society/models.py:57
-msgid "Check off if the worker is still actively working."
-msgstr "Kun kryss av om arbeideren fortsatt jobber aktivt."
-
-#: spbm/apps/society/models.py:60
+#: spbm/apps/society/models.py:55
 msgid "Full name, with first name first."
 msgstr "Fullt navn, med fornavnet først."
 
-#: spbm/apps/society/models.py:62
+#: spbm/apps/society/models.py:57
 msgid "address"
-msgstr "addresse"
+msgstr "adresse"
 
-#: spbm/apps/society/models.py:63
+#: spbm/apps/society/models.py:58
 msgid "Full address including postal code and area."
 msgstr "Full bostedsadresse, inkl. postkode og område."
 
-#: spbm/apps/society/models.py:66
+#: spbm/apps/society/models.py:62
 msgid "account no."
-msgstr "kontonr."
+msgstr "kontonummer"
 
-#: spbm/apps/society/models.py:67
+#: spbm/apps/society/models.py:63
 msgid "Norwegian bank account number, no periods, 11 digits."
 msgstr "Norsk bankkontonummer, uten punktum, 11 siffer."
 
-#: spbm/apps/society/models.py:70
+#: spbm/apps/society/models.py:68
 msgid "person ID"
 msgstr "fødselsnummer"
 
-#: spbm/apps/society/models.py:74
+#: spbm/apps/society/models.py:72
 msgid "norl&oslash;nn number"
 msgstr "norlønn-nummer"
 
-#: spbm/apps/society/models.py:75
+#: spbm/apps/society/models.py:73
 msgid "Employee number in the wage system, Norl&oslash;nn. <strong>Must</strong> exist and be correct!"
 msgstr "Asattnummeret i lønnssystemet Norlønn. <strong>Må</strong> eksistere og være korrekt satt opp!"
 
-#: spbm/apps/society/models.py:91 spbm/apps/society/models.py:180
+#: spbm/apps/society/models.py:98
+msgid "active"
+msgstr "aktiv"
+
+#: spbm/apps/society/models.py:99
+msgid "Check off if the worker is still actively working."
+msgstr "Kun kryss av om arbeideren fortsatt jobber aktivt."
+
+#: spbm/apps/society/models.py:108 spbm/apps/society/models.py:218
 msgid "invoice"
 msgstr "faktura"
 
-#: spbm/apps/society/models.py:92
+#: spbm/apps/society/models.py:109
 msgid "invoices"
 msgstr "fakturaer"
 
-#: spbm/apps/society/models.py:102
+#: spbm/apps/society/models.py:124
 msgid "invoice number"
 msgstr "fakturanummer"
 
-#: spbm/apps/society/models.py:104
+#: spbm/apps/society/models.py:126
 msgid "period"
 msgstr "periode"
 
-#: spbm/apps/society/models.py:106
+#: spbm/apps/society/models.py:128
 msgid "invoice paid"
 msgstr "faktura er betalt"
 
-#: spbm/apps/society/models.py:108
+#: spbm/apps/society/models.py:130
 msgid "User which closed the period the invoice belongs to."
 msgstr "Bruker som lukket perioden som fakturaen er en del av."
 
-#: spbm/apps/society/models.py:147
+#: spbm/apps/society/models.py:166
+#, fuzzy
```

Comment:
Alle oversettelser med `fuzzy` [må gjennomgås og bekreftes som korrekt](https://www.gnu.org/software/gettext/manual/html_node/Fuzzy-Entries.html). De er som regel, eller alltid i vårt tilfelle, verktøyet som foreslår noen strenger for _delvis_ nye `msgid`-er, men som trenger en bekreftelse på at det er rett.

Reply:
Jeg bare lot det stå `fuzzy` alle stedene det fantes fra før. Du foreslår at jeg manuelt fjerner alle `fuzzy` dersom jeg mener oversettelsen stemmer?